src/Framework/FieldsAPI/Concerns/HasSetters.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Give\Framework\FieldsAPI\Concerns;
+
+use BadMethodCallException;
+
+/**
+ * Trait HasGettersAndSetters
+ *
+ * @unreleased
+ */
+trait HasSetters
+{
+    public function __call($name, $arguments)
+    {
+        $prefix = substr($name, 0, 3);
+        $property = lcfirst(substr($name, 3));
+
+        if (!property_exists($this, $property)) {
+            throw new BadMethodCallException(sprintf(__('Property %s does not exist', 'givewp'), $name));
+        }
+
+        if ($prefix === 'set') {
```

Comment:
The `set` prefix isn't necessary here. We have already established a convention for setters in the Fields API without the prefix.